### DISCUSSION TITLE
Not able to uninstall nor update the app.

### DISCUSSION THREAD

**Discussion by macmus82** [kyleneideck/BackgroundMusic#740] [General]
What is going on ?

it fails to update using brew update command 

==> Removing launchctl service com.bearisdriving.BGM.XPCHelper
Password:
==> Uninstalling packages with sudo; the password may be necessary:
==> Removing files:
/Library/Application Support/Background Music
/Library/Audio/Plug-Ins/HAL/Background Music Device.driver
/usr/local/libexec/BGMXPCHelper.xpc
Could not kickstart service "com.apple.audio.coreaudiod": 1: Operation not permitted
==> Purging files for version 0.4.3 of Cask background-music
Error: background-music: Failure while executing; `/usr/bin/sudo -E -- /bin/launchctl kickstart -kp system/com.apple.audio.coreaudiod` exited with 1. Here's the output:
Could not kickstart service "com.apple.audio.coreaudiod": 1: Operation not permitted

**Comment by kyleneideck** [comment]
Looks related to this issue: https://github.com/kyleneideck/BackgroundMusic/issues/727
